src/sybil/column_store_test.go

```diff
@@ -184,7 +184,7 @@ func TestBigIntColumns(t *testing.T) {
 
 	unloadTestTable(tableName)
 
-	nt.Options.SAMPLES = true
+	OPTS.SAMPLES = true
```

Comment:
what's going on with OPTS? still exists or not?

Reply:
I had intended on removing it completely, I'll revise and do so.